monkey/monkey_island/cc/services/zero_trust/test_common/monkey_finding_data.py

```diff
@@ -0,0 +1,33 @@
+from monkey_island.cc.models.zero_trust.event import Event
+from monkey_island.cc.models.zero_trust.monkey_finding_details import MonkeyFindingDetails
+
+EVENTS = [
+    {
+        "timestamp": "2021-01-20T15:40:28.357Z",
+        "title": "Process list",
+        "message": "Monkey on gc-pc-244 scanned the process list",
```

Comment:
Better not use guardicore hostnames in doc

Reply:
Maybe it's GoogleCloud, not GuardiCore :P

Reply:
Let's change this anyways